src/ui/public/styles/variables/colors.less

```diff
@@ -0,0 +1,70 @@
+//== Colors
+//
+//## Gray and brand colors for use across Bootstrap.
+
+@white:                                 #ffffff;
+@black:                                 #000000;
+
+@gray2:                                 #222222;
+@gray3:                                 #333333;
+@gray4:                                 #444444;
+@gray5:                                 #555555;
+@gray6:                                 #666666;
+@gray7:                                 #777777;
+@gray8:                                 #888888;
+@gray9:                                 #999999;
+@gray10:                                #aaaaaa;
+@gray11:                                #bbbbbb;
+@gray12:                                #cccccc;
+@gray13:                                #dddddd;
+@gray14:                                #eeeeee;
+
+@gray-base:                             @black;
+@gray-darker:                           @gray2;
+@gray-dark:                             #7b8a8b;
+@gray:                                  #95a5a6;
+@gray-light:                            #b4bcc2;
+@gray-lighter:                          #ecf0f1;
+
+@brand-primary:                         @gray4;
+@brand-success:                         #31C471;
+@brand-info:                            #1F6B7A;
+@brand-warning:                         #F39C12;
+@brand-danger:                          #E74C3C;
+
+//== Tables
+//** Background color used for `.table-striped`.
+@table-bg-accent:                       #f9f9f9;
```

Comment:
I actually agree with you here, but I think that instead of hardcoding a value to the `table-bg-accent` variable, we should use one of the existing colors. That's what would make sense to me. And if not, then maybe it makes more sense to define a variable here for the color itself, and use it when specifying the `table-bg-color` in the other file. Does that make sense?